crates/optimism/node/src/txpool.rs

```diff
@@ -46,6 +46,13 @@ pub struct OpPooledTransaction {
     inner: EthPooledTransaction<OpTransactionSigned>,
     /// The estimated size of this transaction, lazily computed.
     estimated_tx_compressed_size: OnceLock<u64>,
+
+    /// Optional conditional attached to this transaction. Is this
+    /// needed if this field is on OpTransactionSigned?
+    conditional: Option<TransactionConditional>,
```

Comment:
yes we need this field here.

this option is also fine because this tx type is always wrapped in an Arc so it's always on the heap anyway